packages/component/src/Utils/activityAltText.ts

```diff
@@ -108,22 +101,11 @@ export default function activityAltText(
     return false;
   }
 
-  const { speak } = activity;
-
-  if (speak === '') {
-    // We will treat the activity as presentational and skip narrating it.
-    return false;
-  }
-
-  if (typeof speak === 'string') {
-    if (isSSML(speak)) {
```

Comment:
why is isSSML being deleted?

Reply:
if the channelData is unavailable, doesn't the SSML still need to be parsed?

Reply:
We are not using SSML (i.e. `speak` field) to compute the text alternative anymore.